packages/localize/README.md

```diff
@@ -3,187 +3,42 @@
-And this is there the second option comes in handy.
-
-1. Using the loaders preconfigured via `setupNamespaceLoader()`:
-
-```js
-// using the regexp to match all component names staring with 'my-'
-localize.setupNamespaceLoader(/my-.+/, async (locale, namespace) => {
-  const response = await fetch(`https://api.example.com/?namespace=${namespace}&locale=${locale}`);
-  return response.json();
-});
-
-await Promise.all([
-  localize.loadNamespace('my-hello-component');
-  localize.loadNamespace('my-goodbye-component');
-]);
-```
-
-Thus there is a loader function for all components having a certain prefix in a name.
-
-### Page locale
-
-The single source of truth for page locale is `<html lang="my-LOCALE">`.
-Accessing it via `localize.locale` getter/setter is preferable.
-
-External tools like Google Chrome translate or a demo plugin to change a page locale (e.g. in the Storybook) can directly change the attribute on the `<html>` tag.
-
-If `<html lang>` is empty the default locale will be set to `en-GB`.
-
-#### Changing the page locale
-
-Changing the locale is simple yet powerful way to change the language on the page without reloading it:
-
-```js
-localize.addEventListener('localeChanged', () => {
-  // do smth when data are loaded for a new locale
-});
-
-// change locale (syncs to `<html lang="es-ES">` and fires the event above)
-localize.locale = 'es-ES';
-```
-
-If the locale is changed when a few namespaces have been already loaded for the previous one, all the data will be requested for existing namespaces for a new locale and only after that the event listeners will be called.
-This ensures that all data necessary for localization is loaded prior to rendering.
-If a certain namespace for a certain locale has been loaded previously, it will never be fetched again until the tab is reloaded in the browser.
-
-#### Fallback locale
-
-Due to the need to develop application code with not everything translated (yet) to all languages, it is good to have a fallback.
-By default the fallback is `en-GB`, meaning that if some namespace does not have data for the current page locale, the `en-GB` will be loaded, making it an important foundation for all other locales data.
-In addition to that the fallback is a necessary mechanism to allow some features of the browsers like Google Chrome translate to work and use the same original data for translations into all not officially supported languages.
-
-> We highly discourage using fallback locales as a user, because it is bug-prone with things like date and number formatting. Please use full locales whenever possible.
-
-### Rendering data
-
-When all necessary data are loaded and you want to show localized content on the page you need to format the data.
-`localize.msg` comes into play here.
-It expects a key in the format of `namespace:name` and can also receive variables as a second argument.
+Rendering data:
```

Comment:
Maybe makes sense to show at least a quick `litMsg` example

Reply:
imho `litMsg` should only be used if you really know what you are doing... 
can be added to deep dive or so...